### PR TITLE
Task/android 11 fixes

### PR DESCRIPTION
**Pull request by baldermork**
Bump language level, API level, support library++, and add check for missing cache directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,13 +82,13 @@ repositories {
 }
 
 dependencies {
-    compile('no.finntech:capturandro:0.0.26@aar')
+    compile('no.finntech:capturandro:1.5@aar')
```
Comment (baldermork) [inline README.md:85]:
```suggestion
    compile('no.finntech:capturandro:1.0.5@aar')
```

File: library/src/main/java/no/finntech/capturandro/BitmapUtil.java [hunk 2]
```diff
@@ -54,16 +55,23 @@ static Uri getProcessedImage(Context context, InputStream inputStream, int longe
 
     @RequiresPermission(Manifest.permission.WRITE_EXTERNAL_STORAGE)
     static String getUniqueFilename(Context context) {
-        String parent = context.getExternalCacheDir().getAbsolutePath();
-        /**
+        File cacheDir = getCacheDir(context);
+        return new File(cacheDir, "capturandro-" + System.currentTimeMillis() + "." + random.nextInt() + ".jpg").getAbsolutePath();
```
Comment (kvitvik) [inline library/src/main/java/no/finntech/capturandro/BitmapUtil.java:59]: Maybe the `capturandro-` prefix should be a constant?

File: library/src/main/java/no/finntech/capturandro/Capturandro.java [hunk 3]
```diff
@@ -89,7 +88,6 @@ public static String getFileProviderAuthority() {
      * When executing subscribe on the Obserable<Uri> image processing may take place (on a seperate thread).
      * Image processing is done on a single background thread to prevent oom
      */
-    @RequiresPermission(allOf = { Manifest.permission.WRITE_EXTERNAL_STORAGE, Manifest.permission.CAMERA })
```
Comment (kvitvik) [inline library/src/main/java/no/finntech/capturandro/Capturandro.java:92]: Maybe we should have kept the permission annotation?
`@RequiresPermission(Manifest.permission.CAMERA)`

(In that case, same with the function that calls this one)

**Review by kvitvik — APPROVED** [review]
Oh my god, you killed `AsyncTask`. You bastard!

LGTM 👍 Just one question about the camera permission.

**Pull request merged**